src/components/skills-quiz/SkillsQuizStepper.jsx

```diff
@@ -344,7 +392,13 @@ const SkillsQuizStepper = ({ isStyleAutoSuggest }) => {
               Go back
             </Button>
             <Stepper.ActionRow.Spacer />
-            <Button onClick={() => closeSkillsQuiz()}>Done</Button>
+            <Button onClick={() => closeSkillsQuiz()}>
+              <FormattedMessage
+                id="enterprise.skills.quiz.v1.done.button.label"
+                defaultMessage="Done"
+                description="Done button label on the skills quiz v1 page."
```

Comment:
What will happen on Done? Can we add little bit more context in the description?